src/backend/commands/prepare.c

```diff
@@ -706,7 +706,16 @@ ExplainExecuteQuery(ExecuteStmt *execstmt, IntoClause *into, ExplainState *es,
 		PlannedStmt *pstmt = (PlannedStmt *) lfirst(p);
 
 		if (IsA(pstmt, PlannedStmt))
+		{
+			/*
+			* GPDB_92_MERGE_FIXME: it really should be an optimizer's responsibility
+			* to correctly set the into-clause and into-policy of the PlannedStmt.
+			*/
+			if (into != NULL)
+				pstmt->intoClause = copyObject(into);
```

Comment:
Why not just do it in `ExplainOnePlan`? And remove it from `ExplainOneQuery`. To avoid duplicating the code.

Reply:
moved